libsolidity/Parser.cpp

```diff
@@ -692,6 +694,39 @@ vector<ASTPointer<Expression>> Parser::parseFunctionCallArguments()
 	return arguments;
 }
 
+void Parser::parseFunctionCallArguments(vector<ASTPointer<Expression>>& _arguments, vector<string>& _names)
+{
+	Token::Value token = m_scanner->getCurrentToken();
+	if (token == Token::LBRACE)
+	{
+		// call({arg1 : 1, arg2 : 2 })
+		expectToken(Token::LBRACE);
+		while (m_scanner->getCurrentToken() != Token::RBRACE)
+		{
+			string identifier = *expectIdentifierToken();
+			expectToken(Token::COLON);
+			ASTPointer<Expression> expression = parseExpression();
+
+			_arguments.push_back(expression);
+			_names.push_back(identifier);
+
+			if (m_scanner->getCurrentToken() == Token::COMMA)
+			{
```

Comment:
This and the following two pairs of braces should go according to the coding style.